test/images/volumes-tester/gluster/Makefile

```diff
@@ -12,7 +12,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-TAG = 0.4
+TAG = 0.5
```

Comment:
there's apparently already a `gcr.io/google-containers/volume-gluster:0.5`, which is exactly the same as 0.4. I guess I'll just push over it.

```
DIGEST        TAGS            TIMESTAMP
5c3e91a365e7  0.4,0.5,latest  2017-01-25T11:02:27
```